tools/taskcluster/execute-firebase-test.sh

```diff
@@ -6,28 +6,49 @@
 # This script installs google cloud sdk, logs into google firebase, executes tests,
 # and collects test artifacts into the test_artifacts folder
 
-# If a command fails then do not proceed and fail this script too.
-set -ex
```

Comment:
I removed it because tools/taskcluster/google-firebase-testlab-login.sh has set -ex, so thought it would be redundant

Reply:
We are setting this flag so that **this** script should stop. I don't think it will be affected from flags in other scripts - but I do not know fore sure.